src/bentoml/_internal/context.py

```diff
@@ -92,16 +92,21 @@ def from_http(request: "starlette.requests.Request") -> "InferenceApiContext":
     class RequestContext:
         metadata: Metadata
         headers: Metadata
+        query_params: Metadata
 
-        def __init__(self, metadata: Metadata):
+        def __init__(self, metadata: Metadata, query_params: Metadata):
             self.metadata = metadata
             self.headers = metadata
+            self.query_params = metadata
```

Comment:
This line should be `self.query_params = query_params`

Reply:
Indeed :facepalm:. Adding a test for this.